<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FocalDialog</class>
 <widget class="QDialog" name="FocalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="3" column="0">
    <widget class="QGroupBox" name="principal_point">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Principal Point</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>X offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="principalOffsetX"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Y offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="principalOffsetY"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="lens_distortion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Lens distortion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>K1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="k1"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>K2:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="k2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>P1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="p1"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>P2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="p2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Resolution</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="width">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="height">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Focal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Focal:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="focalLength">
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>CCD width:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="ccdWidth"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>CCD height:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="ccdHeight"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="equivalent">
        <property name="text">
         <string>Equivalent 35mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="real">
        <property name="text">
         <string>Real (mm)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FocalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FocalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
